static/scripts/topicEdit.js

```diff
@@ -7,6 +7,7 @@ import {
 } from 'react-sortable-hoc';
 import './calendar';
 import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import shortid from 'shortid';
```

Comment:
I think `shortid` is deprecated and should not be used anymore (https://www.npmjs.com/package/shortid). The authors of shortid recommend 'nanoid' instead. (https://github.com/ai/nanoid/)

Reply:
Yeah, it was deprecated due to the cryptographics reasons. In this case it is just used to generate random id for newly added  editor. I don't care if someone will guess this Id because there is no harm behind it. Currently it is used in some places in our code. I would recommend to create a follow up ticket (I can do it) to replace shortid with nano id what do you say ?

Reply:
Sounds good 👍